app/controllers/sign_in_controller.rb

```diff
@@ -11,6 +11,19 @@ class SignInController < ApplicationController
   REDIRECT_URLS = %w[idme logingov dslogon mhv].freeze
   BEARER_PATTERN = /^Bearer /.freeze
 
+  STATSD_SIS_AUTHORIZE_ATTEMPT_SUCCESS = 'api.sis.auth.success'
```

Comment:
Could we move this into a constants file? Could just a file like `app/services/sign_in/constants/statsd.rb`